sdk/__tests__/utils.ts

```diff
@@ -271,7 +271,7 @@ async function waitForRelay(
   const deliveryHash = keccak256(vaa!.hash);
 
   const wormholeRelayer = IWormholeRelayer__factory.connect(
-    dst.context.config.contracts.relayer!,
+    "0xcC680D088586c09c3E0E099a676FA4b6e42467b4", // dst.context.config.contracts.relayer!,
```

Comment:
fwiw this could also be overridden here:

https://github.com/wormhole-foundation/example-native-token-transfers/blob/b547154bbf67792d6ed28f3fa1cf724d90fc1006/sdk/__tests__/utils.ts#L79-L86